geniza/locale/ar/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,62 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-18 16:44-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
```

Comment:
Are these configurable somewhere in our settings? Doesn't matter for the experiment, just curious.

Reply:
Although it would be good to know which fields translators are supposed to update when they work on translations.

Reply:
I'm not sure where they come from...as far as I can tell they're defaults from gnu gettext, which django uses under the hood. I can see if there's some way to configure them, but for now at least if we edit them they won't be clobbered when we regenerate the file.

Reply:
As far as I can tell, translators only ever touch the `msgstr` field...maybe there are some docs somewhere we can point people to, since it's a common utility.

Reply:
it looks like if we want to customize what `gettext()` is doing (e.g. to override the `.pot` template used to generate the `.po` files), we would need to [create our own management command](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#customizing-the-makemessages-command) that extends `makemessages`.